pkg/controller/service/service_controller.go

```diff
@@ -264,7 +264,9 @@ func (s *ServiceController) createLoadBalancerIfNeeded(key string, service *v1.S
 		needDelete := true
 		_, exists, err := s.balancer.GetLoadBalancer(s.clusterName, service)
 		if err != nil {
-			return fmt.Errorf("Error getting LB for service %s: %v", key, err), retryable
+			err = fmt.Errorf("Error getting LB for service %s: %v", key, err)
```

Comment:
this will make the event text stutter.  the event is already about a particular service so including the name of the service again is redundant.

Reply:
Indeed, will fix these, thanks for the review 😉